### ISSUE TITLE
Mirror View

### ISSUE THREAD

**Issue by dimovskidamjan** [zhxnlai/ZLHistogramAudioPlot#3]
There is a shouldMirror property, but it doesn't seem to be working, because I've set it to "YES" and the histogram isn't mirrored.

**Comment by zhxnlai** [comment]
Do you mean [this](https://github.com/syedhali/EZAudio/blob/5215465565ab6c97fe6e82c596e68a9b140ace72/EZAudio/EZPlot.h#L92) property? The implementation of ZLHistogramAudioPlot is rather different from that of EZAudioPlot and it does not make use of `shouldMirror`.

**Comment by dimovskidamjan** [comment]
Yes, that's exactly what I meant. I realized that it doesn't use it, or rather I found out by going through the code. What I'm saying is that something similar should be implemented for ZLHistogramAudioPlot as well.
